feature/qos/ate_tests/one_sp_queue_traffic_test/one_sp_queue_traffic_test.go

```diff
@@ -777,38 +777,70 @@ func ConfigureQoS(t *testing.T, dut *ondatra.DUTDevice) {
 	d := &oc.Root{}
 	q := d.GetOrCreateQos()
 
+	type qosVals struct {
+		be0, be1, af1, af2, af3, af4, nc1 string
+	}
+
+	qos := qosVals{
+		be0: "BE0",
+		be1: "BE1",
+		af1: "AF1",
+		af2: "AF2",
+		af3: "AF3",
+		af4: "AF4",
+		nc1: "NC1",
+	}
+
+	if dut.Vendor() == ondatra.JUNIPER {
+		qos = qosVals{
+			be0: "6",
+			be1: "0",
+			af1: "4",
+			af2: "1",
+			af3: "5",
+			af4: "2",
+			nc1: "3",
+		}
+
+	}
+
+	nc1InputWeight := uint64(200)
+	if deviations.SchedulerInputWeightLimit(dut) {
+		nc1InputWeight = uint64(100)
```

Comment:
Updated the explanation in description and deviation description

Reply:
In this test, since there is only one queue assotiated with scheduler of sequence(0), the weight is irrelevant. This is because scheduler 0 is served 1st as highest strict priority.
Therefore I would recomend to set it to any value that is common for all implemetations. That is int(100). And remove deviation.

Reply:
@rszarecki +1: We can update this to 100 and remove the attachment of deviation flag here. But we will add the deviation to deviation.go file for DP-1.4 as that has multiple queues w/ different weights.